.travis.yml

```diff
@@ -15,18 +15,18 @@ env:
 
 matrix:
   include:
-    - python: "2.7"
-      dist: trusty
-      env: DJANGO=1.11 COVERAGE="coverage run -m" TEST_K="not ddns_client" TEST_OPTS="--pep8"
     - python: "3.5"
       dist: trusty
-      env: DJANGO=1.11
+      env: DJANGO=2.2
     - python: "3.6"
       dist: trusty
```

Comment:
if the tests do not work on trusty (, we maybe could move 3.5 and 3.6 this to xenial?